src/Umbraco.Web/Unversion/IUnversionService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Umbraco.Core.Models;
+
+namespace Umbraco.Web.Unversion
```

Comment:
Why is this in Web? It has nothing to do with Web and doesn't require only 'Web' things